api-auto-registration/troubleshooting.hbs.md

```diff
@@ -72,6 +72,10 @@ This section includes commands for debugging or troubleshooting the APIDescripto
     > **Note** If you manually remove the finalizers from the APIDescriptor resources, you can have
     > stale API entities within Tanzu Developer Portal that you must manually deregister.
 
+1. Fix a `CuratedAPIDescriptor` not matching with a SCG.
+
+    This could happen if the groupId and version of the `CuratedAPIDescriptor` does not match any available SCG.
```

Comment:
Shall we add a sentence or two to make suggestions to this error case?
```suggestion
    This could happen if the groupId and version of the `CuratedAPIDescriptor` does not match any available `SpringCloudGateway` resource.
    You can remove the `"apis.apps.tanzu.vmware.com/route-provider": "spring-cloud-gateway"` annotation from your `CuratedAPIDescriptor` to skip SCG matching, or make sure you have a matching SCG applied to the cluster. 
```